controllers/addinvoice.ctrl.go

```diff
@@ -37,6 +37,7 @@ func (controller *AddInvoiceController) AddInvoice(c echo.Context) error {
 
 func AddInvoice(c echo.Context, svc *service.LndhubService, userID int64) error {
 	var body AddInvoiceRequestBody
+	limits := svc.GetLimitsFromContext(c)
```

Comment:
There is no need to already do `GetLimitsFromContext` here. We have the context all the way down, we can just fetch the limits when we need them, there is no need to change the arguments of all these functions